LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2019 Shreyas Nagare
+Copyright 2020 Shreyas Nagare
```

Comment:
You don't have copyright on this. Change it to `Brick Development Team` like the Brick license